src/UI/pages/user_settings.tsx

```diff
@@ -30,94 +31,12 @@ function UserSettings() {
-                  <Grid container>
-                    <Grid item xs={12} md={6}>
-                      <div>
-                        Welcome <b>{user?.nickname}</b> !
-                      </div>
-                    </Grid>
-                    {/* <Grid item xs={12} md={6}>
-                                        <Box sx={{ '& > :not(style)': { m: 1 } }}>
-                                            <Fab color="primary" aria-label="edit">
-                                                <EditIcon />
-                                            </Fab>
-                                            <Fab color="error" aria-label="add">
-                                                <DeleteForeverIcon />
-                                            </Fab>
-                                        </Box>
-                                    </Grid> */}
-                  </Grid>
-                </Grid>
-                <Grid item sm={12} md={6}>
-                  <div>
-                    <h4 color="primary">Personal information</h4>
-                    <div style={{ marginTop: 30 }}>
-                      <TextField
-                        id="outlined-basic"
-                        label="Name"
-                        variant="outlined"
-                        value={user?.name}
-                        fullWidth={true}
-                      />
-                    </div>
-                    <div style={{ marginTop: 30 }}>
-                      <TextField
-                        id="outlined-basic"
-                        label="Email"
-                        variant="outlined"
-                        value={user?.email}
-                        fullWidth={true}
-                      />
-                    </div>
-                  </div>
-                </Grid>
-                <Grid item sm={12} md={6}>
-                  <div>
-                    <h4 color="primary">Access information</h4>
-                    <div>
-                      <List
-                        sx={{
-                          width: '100%',
-                          maxWidth: 360,
-                          bgcolor: 'background.paper',
-                        }}
-                      >
-                        {userRoles.map((role, index) => (
-                          <ListItem key={index}>
-                            <ListItemAvatar>
-                              <Avatar>
-                                <LockOpenIcon />
-                              </Avatar>
-                            </ListItemAvatar>
-                            <ListItemText primary={role} />
-                          </ListItem>
-                        ))}
-                      </List>
-                    </div>
-                  </div>
-                </Grid>
-              </Grid>
-            </SectionPanel>
-          </Container>
+          <UserSettingForm user={user} userRoles={userRoles} />
         </main>
       </div>
     );
   } else {
```

Comment:
Do we need this if statement? If a user isn't logged in, they should be redirected to the login page as in line26, yes?